.github/workflows/upload_google_play.yml

```diff
@@ -23,36 +23,36 @@ jobs:
       - uses: actions/setup-java@v1
         with:
           java-version: 8
-      - name: Build core and features
+      - name: Build API and features
```

Comment:
Naming nit: API *server*? API *backend*? The API is just the interface, as its name implies.

Reply:
It's called "Feature API" -> Given how this naming is only used in workflows I think it's so easy to change later, we could :P